Dockerfile

```diff
@@ -35,11 +35,11 @@ RUN for dir in \
       /runner/inventory \
       /runner/project \
       /runner/artifacts ; \
-    do mkdir -m 0775 -p $dir ; chmod g+rw $dir ; chown -R runner:runner $dir ; done && \
+    do mkdir -m 0775 -p $dir ; chmod g+rwx $dir ; chgrp root $dir ; done && \
```

Comment:
Still processing things myself, I now kind of see how this makes the layers closer to:

```
RUN chgrp -R 0 /some/directory && \
    chmod -R g=u /some/directory
```

from https://docs.openshift.com/container-platform/3.3/creating_images/guidelines.html, with a notable difference being the named `root` group, which was created a few lines above.